### PR TITLE
Fixed cache_retention granularity.

### PR DESCRIPTION
**Pull request by p0psicles**
- [ ] PR is based on the DEVELOP branch
- [ ] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review
- [ ] Read [contribution guide](https://github.com/PyMedusa/SickRage/blob/master/.github/CONTRIBUTING.md)

* Should be cast to int, as we want to step by day, not tenth of day.

@labrys ?
I've changed it because else in config you get to see 30,0. While stepping did change, saving did not.

### CODE REVIEW

File: sickbeard/server/web/config/search.py [hunk 1]
```diff
@@ -69,7 +69,7 @@ def saveSearch(self, use_nzbs=None, use_torrents=None, nzb_dir=None, sab_usernam
         sickbeard.BACKLOG_DAYS = try_int(backlog_days, 7)
 
         sickbeard.CACHE_TRIMMING = config.checkbox_to_value(cache_trimming)
-        sickbeard.MAX_CACHE_AGE = float(max_cache_age)
+        sickbeard.MAX_CACHE_AGE = try_int(max_cache_age)
```
Comment (medariox) [inline sickbeard/server/web/config/search.py]: Should a non-int default to 0 or 1?

Comment (labrys) [inline sickbeard/server/web/config/search.py]: 0 since in cache trim i do `if days:` so that it would skip the trim instead of trimming all but 1 day.

Comment (labrys) [inline sickbeard/server/web/config/search.py]: Also as this is set by the UI, it should be `int`, `try_int` is unnecessary.

Comment (p0psicles) [inline sickbeard/server/web/config/search.py]: hmm, then we should change those everywhere. As everywhere try_int is used. Also for ex. BACKLOG_DAYS and USENET_RETENTION.

**Comment by p0psicles** [comment]
Approved

**Pull request merged**